diff --git a/dist/rollup.js b/dist/rollup.js
index e128b61b9558318b9f86dc11d72c31f09a8eb7db..01b925934579b757a36dd0fe3e05020ea641d7e0 100644
--- a/dist/rollup.js
+++ b/dist/rollup.js
@@ -6834,7 +6834,7 @@ var getRollupConfig = async (options) => {
           tsconfig: options.tsconfig,
           compilerOptions: {
             ...compilerOptions,
-            baseUrl: compilerOptions.baseUrl || ".",
+            ...(compilerOptions.baseUrl !== undefined ? { baseUrl: compilerOptions.baseUrl } : {}),
             // Ensure ".d.ts" modules are generated
             declaration: true,
             // Skip ".js" generation
